employee_dialog/lib/main.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:employee/Employee profile/employee_preferences.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'Employee profile/profile_page.dart';
+
+void main() async {
```

Comment:
we don't need the async marker on the main method as well. You only need it when you handle http requests